apps/studio/components/interfaces/DataWarehouse/WarehouseAccessTokens.tsx

```diff
@@ -0,0 +1,232 @@
+import React from 'react'
```

Comment:
for consistency:
we separate external imports and internal imports with a linespace, e.g:

```
import { useState } from 'react'
import { useRouter } from 'next/router'

import { useParams } from 'common'
import { FormHeader } from 'components/ui/Forms'
```

Reply:
and then we sort them via command shift o